arrow/arrow.py

```diff
@@ -1122,6 +1122,7 @@ def humanize(
         locale: str = DEFAULT_LOCALE,
         only_distance: bool = False,
         granularity: Union[_GRANULARITY, List[_GRANULARITY]] = "auto",
+        dynamic: bool = False,
```

Comment:
Is this False by default to avoid a breaking change? That's understandable, but also disappointing since to me, the dynamic behaviour seems much more useful than outputting a bunch of zeros for units.

Reply:
@MarkKoz, yes we left as False by default in order to avoid changing the behaviour of humanize drastically. I agree that it would make more sense to leave it as True by default however. @jadchaar @krisfremen @systemcatch what are your thoughts?

Reply:
At the least, I hope changing this can be considered for the next major release (assuming you're following SemVer).

Unrelated: `dynamic` isn't a good name — it's vague and non-self-descriptive. `omit_zeros` or something similar would be clearer.

Reply:
I think we should leave it as False for the time being, do a warning for changing behavior and change it to default True after a few versions.

Reply:
That sounds reasonable. Any thoughts on my name suggestion?

Reply:
Errr throwing out a few ideas for the name, `only_natural`, `minimal`, `drop_zeros`. `omit_zeros` is fine as well.

Reply:
I think omit_zeros is probably the best name

Reply:
Personally, I think the `dynamic` naming is fine, as IMO it indicates it will use dynamically any of the granularity fields that are specified, as the time progresses or is shifted.

Although, omit_zeroes is a nice alternative name, I would prefer dynamic.